Source/monster.cpp

```diff
@@ -4399,8 +4399,6 @@ void FreeMonsters()
 			}
 		}
 	}
-
-	FreeMissiles2();

```

Comment:
This will cause the game not to unload missiles that where only used on a specific floor (monster projectiles). Please restore it in some form :)

Reply:
All missiles will be unloaded here alongside every other missile:
- https://github.com/diasurgical/devilutionX/pull/2796/commits/7f72da754005c826b767cf0a99eac607c620fe4d#diff-84e63ba430a13546a287665b7640118b2f9d66fb9586056bfbab8215bab319c0R1524

Reply:
That's game shutdown, not level transition.

Reply:
The place I pointed to is currently the only place in the codebase where `FreeMonsters` is called.

I think you are mistaken here.

Reply:
`FreeMissiles2()` had exactly one use in `FreeMonsters()`, which itself has exactly one use in `FreeGameMem()`. `FreeGameMem()` gets called on all forms of level transition, starting a new game, quitting diablo, and loading a game.

From what I could tell nothing in `FreeMonsters()` requires the non-monster missile graphics to be freed first so it's safe to unload all missile graphics in one pass.

Reply:
Hmm maybe it would be worth fixing this as some point, it sound like this will mean that more data gets loaded if you play multiple levels in one sitting then if you restart between each level.